spec/services/find/cycle_timetable_spec.rb

```diff
@@ -50,6 +50,22 @@ module Find
       end
     end
 
+    describe '.find_opens(year)' do
+      it 'returns find_opens date for 2021' do
```

Comment:
Do we need this test? Isn't it testing the same thing as `'returns find_opens date for 2024'`, below?

Reply:
There's a test for when an argument is passed and when one is not passed